### ISSUE TITLE
Root Effects from Mob's (Specifically Goblin Javelin Units) not broken by damage?

### ISSUE THREAD

**Issue by Itheryn** [echoes-emu/echoes-bt#63]
**Possible Bug:** Root Effects from mobs (specifically goblin javelin units) not breaking on damage % chance
**Result:** The player is rooted for the full duration of effect regardless of damage
**Reason for report:** This is not a thing in current lotro, and im sure it broke on damage in 2007. Possibly mistaken though.

**Information:**
Goblin Javelin Snare (web/root effect) Does not break upon damage, holding you in place for the full duration of time, regardless of being hit or out of combat. Meaning you are rooted for 10+ Seconds.
Was being hit by 5-8 Goblins in the goblin camp west of Noglond (Ered Luin)
Only 1 Javelin unit (so this was not being re-applied)
Was rooted for the full duration, and died within a second of leaving the root. 

I could be wrong here, but I'm fairly sure this used to break on damage at a 25% chance on being hit.
I know it does in current LOTRO and I don't have any memories of this being a thing in oldschool LOTRO, but maybe I just dont remember it correctly. Its something to look at perhaps, not sure.

**Comment by echoes-emu** [comment]
Haven't heard any more reports so probably not a big issue. will investigate this for full releases

**Comment by Stressy23** [comment]
Working as intended, Enemy minions had stronger CC than players.

**Comment by MasterTeal** [comment]
I also think this is WAI

**Comment by woodx319** [comment]
Yeah player roots are strong and go for their duration. buffs debuffs are also tooltipped pretty well things that have details like that would say it in the debuff description if you hover over it. (%break chance, or breaks on damage or something). 

Tldr: bring some friends to goblin town bois
